app/todo/api.py

```diff
@@ -0,0 +1,37 @@
+from fastapi import APIRouter, Depends
+
+from app.depends.auth import get_current_user
+from app.depends.service import get_todo_service
+from app.todo.models import (DeleteTodo, DeleteTodoInfo,
+                             GetTodoInfo, GetTodoInfoList, InsertTodoInfo)
+from app.todo.service import BaseTodoService
+
+app_router = APIRouter()
+
+
+@app_router.get("/todoinfo", response_model=GetTodoInfoList)
+async def getTodoInfoList(
+    user_id: int = Depends(get_current_user), todo_svc: BaseTodoService = Depends(get_todo_service),
+):
+    todoinfo = await todo_svc.getTodoInfoList(user_id)
+    return GetTodoInfoList(data=todoinfo)
+
+
+@app_router.post("/regtodo", response_model=InsertTodoInfo, status_code=201)

```

Comment:
REST에서 action은 HTTP method로 표현하는걸 권장하고 있습니다.
지금 todo API는 기본적인 CRUD 기능만 포함하므로, `POST /todo`로 endpoint를 지정해도 의미상 혼동이 없을 것 같습니다! (P2)
```suggestion
@app_router.post("/todo", response_model=InsertTodoInfo, status_code=201)

```

Reply:
변경완료했습니다!
10fcdca476d40ccf1fd8c9d6a29a30c9f744a525